Uncaught magic.MagicException when uploading specific file

**Environment information**
* MWDB version (from `/about`): 2.7.0
* Installation method:
  - [ ] mwdb.cert.pl service
  - [x] From PyPi (`pip install mwdb-core`)
  - [ ] From docker-compose
  - [ ] Other (please explain)
* Plugins installed: None

**Behaviour the bug (what happened?)**

Uploading the attached file results in an unhandled exception:

```
[ERROR] Thread-39 - service.error_router:76 - Unhandled exception occurred
Traceback (most recent call last):
  File "/opt/mwdb/venv/lib/python3.9/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/opt/mwdb/venv/lib/python3.9/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/opt/mwdb/venv/lib/python3.9/site-packages/flask_restful/__init__.py", line 467, in wrapper
    resp = resource(*args, **kwargs)
  File "/opt/mwdb/venv/lib/python3.9/site-packages/flask_limiter/extension.py", line 1036, in __inner
    return current_app.ensure_sync(obj)(*a, **k)
  File "/opt/mwdb/venv/lib/python3.9/site-packages/flask_limiter/extension.py", line 1036, in __inner
    return current_app.ensure_sync(obj)(*a, **k)
  File "/opt/mwdb/venv/lib/python3.9/site-packages/flask_limiter/extension.py", line 1036, in __inner
    return current_app.ensure_sync(obj)(*a, **k)
  [Previous line repeated 5 more times]
  File "/opt/mwdb/venv/lib/python3.9/site-packages/flask/views.py", line 84, in view
    return current_app.ensure_sync(self.dispatch_request)(*args, **kwargs)
  File "/opt/mwdb/venv/lib/python3.9/site-packages/flask_restful/__init__.py", line 582, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/opt/mwdb/venv/lib/python3.9/site-packages/mwdb/resources/__init__.py", line 47, in endpoint
    return f(*args, **kwargs)
  File "/opt/mwdb/venv/lib/python3.9/site-packages/mwdb/resources/__init__.py", line 31, in endpoint
    return f(*args, **kwargs)
  File "/opt/mwdb/venv/lib/python3.9/site-packages/mwdb/resources/file.py", line 187, in post
    return self.create_object(obj["options"])
  File "/opt/mwdb/venv/lib/python3.9/site-packages/mwdb/resources/object.py", line 125, in create_object
    item, is_new = self._create_object(
  File "/opt/mwdb/venv/lib/python3.9/site-packages/mwdb/resources/file.py", line 34, in _create_object
    return File.get_or_create(
  File "/opt/mwdb/venv/lib/python3.9/site-packages/mwdb/model/file.py", line 103, in get_or_create
    file_type=calc_magic(file_stream),
  File "/opt/mwdb/venv/lib/python3.9/site-packages/mwdb/core/util.py", line 91, in calc_magic
    return magic.maybe_decode(magic.magic_file(magic_cookie, fd_path))
  File "/opt/mwdb/venv/lib/python3.9/site-packages/magic.py", line 272, in magic_file
    return _magic_file(cookie, coerce_filename(filename))
  File "/opt/mwdb/venv/lib/python3.9/site-packages/magic.py", line 209, in errorcheck_null
    raise MagicException(err)
magic.MagicException: b', dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2 error reading (Invalid argument)'
[INFO] Thread-39 - app.log_request:141 - request - path:/api/file - arguments:{} - method:POST - status:500 - response_time:0:00:00.059437 - response_size:37
127.0.0.1 - - [02/May/2022 15:09:10] "POST /api/file HTTP/1.0" 500 -
```

It appears a `magic.MagicException` was thrown which wasn't caught. I have `python-magic 0.4.18` installed.
Running `file` on the input file results in:

```
/tmp/debug.bin: ERROR: , dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2 error reading (Invalid argument)
```

<!-- What actually happened? -->

**Expected behaviour**

mwdb catching the `magic.MagicException` and handling it. For example by
[debug.bin.zip](https://github.com/CERT-Polska/mwdb-core/files/8603057/debug.bin.zip)
 defaulting to `data` as filetype.

<!-- What did you expect to happen? -->

**Reproduction Steps**

<!-- Please explain how can we reproduce your problem. -->
Just uploading the attached file with the `File Upload` option triggers the error.

**Screenshots**

<!-- If applicable, add screenshots to help explain your problem. -->

**Additional context**

Archive password is `infected`.

<!-- Add any other context about the problem here. -->

--- a/mwdb/core/util.py
+++ b/mwdb/core/util.py
@@ -79,7 +79,7 @@ def get_fd_path(stream):
         return None
 
 
-def calc_magic(stream):
+def calc_magic(stream) -> str:
     # Missing python-magic features:
     # - magic_descriptor (https://github.com/ahupp/python-magic/pull/227)
     # - direct support for symlink flag
@@ -93,9 +93,11 @@ def calc_magic(stream):
             # Handle BytesIO in-memory streams
             stream.seek(0, os.SEEK_SET)
             return magic.maybe_decode(magic.magic_buffer(magic_cookie, stream.read()))
+    except magic.MagicException:
+        # If libmagic fails, we fallback to 'data'
+        return "data"
     finally:
         magic.magic_close(magic_cookie)
-    return None
 
 
 def calc_ssdeep(stream):

